tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -899,8 +951,7 @@ def test_column_expectation(
         )
         result = checkpoint.run()
 
-        exc_details = _get_exception_details(result, prettyprint=True)
-        assert not exc_details, exc_details[0]["raised_exception"]
+        _ = _get_exception_details(result, prettyprint=True)
```

Comment:
Stop caring about the implementation detail of if an expectation failed with exceptions or not.